WebUI/Pages/Index.razor

```diff
@@ -1,4 +1,115 @@
 ﻿@page "/"
 @layout UnauthorizedPagesLayout
+@using ConestogaInsidersClub.Data.Models;
+@using ConestogaInsidersClub.Data.DataAccess;
+@using ConestogaInsidersClub.Pages.ViewModels;
 
-<h1>Games List</h1>
+@inject IGameService gameService;
+
+
+<h1>Conestoga Insiders Club</h1>
+<p>Your source of the latest games and events</p>
+
+<hr />
+
+@*--- Display Game List ---*@
+@if (showDetails)
+{
+    <br />
+    <h3>Game Details</h3>
+    <br />
+    <table>
+        <tr>
+            <td>Name: </td>
+            <td class="align-content-end"><span id="Game_Name">@game.Name</span></td>
+        </tr>
+        <tr>
+            <td>Description: </td>
+            <td><span id="Game_Name">@game.Description</span></td>
+        </tr>
+        <tr>
+            <td>Price: </td>
+            <td><span id="Game_Name">@game.Price.ToString("C")</span></td>
+        </tr>
+        <tr>
+            <td>Genre: </td>
+            <td><span id="Game_Name">@game.Genre</span></td>
+        </tr>
+    </table>
+    <br />
+    <AuthorizeView>
+        <Authorized>
+            @*TODO - Hook up game purchase and wishlist features*@
+            <button class="btn btn-primary">Buy Game</button>
+            <button class="btn btn-primary">Add To Wishlist</button>
+            <button class="btn btn-primary">Leave Rating</button>
+        </Authorized>
+    </AuthorizeView>
+    <button class="btn btn-secondary" @onclick="ToggleDetails">Go Back</button>
+
+}
+else if (games != null && games.Count != 0)
+{
+    <br />
+    <h3>Games</h3>
+    <p>Below you can find a selection of our available games.</p>
+    <br />
+    <div class="row" style="padding-top: 10px">
+        @foreach (var game in games)
+        {
+            <div class="col-sm-6">
+                <div class="card">
+                    <div class="card-body">
+                        <p class="card-text float-right">Price: @game.Price.ToString("C")</p>
+                        <h5 class="card-title">@game.Name</h5>
+                        <p class="card-text">@game.Description</p>
+                        <button class="btn btn-primary" @onclick="() => DisplayDetails(game)">View Details</button>
+                    </div>
+                </div>
+            </div>
+        }
+    </div>
+}
+else
```

Comment:
This is quite confusing as you are putting both a single game view and the game listing in the same component. If we later will require to go to a game details page just by navigation (for example from the cart page), that would not be possible as there's no separate page dedicated to a game. I think, that at the very least we should split the Game Listing page and the Game Details page into separate components under separate urls.